TASHealthCheck/src/main/java/com/tas/healthcheck/service/TASApplicationService.java

```diff
@@ -61,6 +64,12 @@ public boolean removeApplicationById(int id) {
 		return tasApplicationDao.removeById(id);
 	}
 
+	
+	private static int STATUS_ERROR = -1;
+	private static int STATUS_OFF = 0;
+	private static int STATUS_UP = 1;
+	private static int STATUS_SOME = 2;
+	private static int STATUS_DOWN = 3;
```

Comment:
For the sake of sorting I would like this logic to look something like:
DOWN=3
SOME=2
ERROR=1
UP=0
OFF=-1